src/parsers/babel_parser.js

```diff
@@ -161,7 +161,7 @@ export const parse = (file: string, data: ?string, options: ?parser.ParserOption
         isFunctionDeclaration(element.expression.right.type)
       ) {
         searchNodes(element.expression.right.body, parent);
-      } else if (element.type === 'ReturnStatement' && element.argument.arguments) {
+      } else if (element.type === 'ReturnStatement' && element.argument && element.argument.arguments) {
```

Comment:
`else if (element.type === 'ReturnStatement' && element.argument?.arguments)`

you could use optional chaining here instead.
even if its still javascript it's already used a few times in this file.

Reply:
good point